client/app/nasha/partition/custom-snapshot/add/nasha-partition-custom-snapshot-add.html

```diff
@@ -1,82 +1,38 @@
 
-        <div class="oui-modal__body oui-modal__body_no-icon">
-            <h2 class="oui-modal__title"
-                data-translate="nasha_snapshot_of"
-                data-translate-values="{ partitionName: NashaPartitionCustomSnapshotAddCtrl.data.partition.partitionName }">
-            </h2>
+        <oui-message type="info">
+            <span data-translate="nasha_custom_snapshot_modal_informations_size"></span>
+        </oui-message>
 
-            <p>
-                {{ :: 'nasha_custom_snapshot_modal_title' | translate }}
-                <i class="fa fa-question-circle fs18 left-space-m4"
-                data-uib-tooltip="{{ :: 'nasha_custom_snapshot_modal_informations' | translate }}"
-                data-tooltip-placement="right"></i>
-            </p>
+        <h2 class="oui-modal__title"
+            data-translate="nasha_snapshot_of"
+            data-translate-values="{ partitionName: NashaPartitionCustomSnapshotAddCtrl.data.partition.partitionName }">
+        </h2>
 
-            <oui-message type="info">
-                <span data-translate="nasha_custom_snapshot_modal_informations_size"></span>
-            </oui-message>
-
-            <div class="oui-field"
-                data-ng-class="{ 'oui-field_error': nashaCustomSnapshotAddForm.inputSnapshotName.$invalid }">
-                <label data-translate="nasha_custom_snapshot_modal_label_snapshot_name"
-                    for="inputSnapshotName"
-                    class="oui-field__label oui-label"></label>
-                <div class="oui-field__content">
-                    <div class="oui-field-control">
-                        <div class="oui-input-group">
-                            <button class="oui-button oui-button_small-width"
-                                disabled
-                                type="button"
-                                data-ng-bind="::NashaPartitionCustomSnapshotAddCtrl.data.customSnapshot.prefix"></button>
-                            <input type="text"
-                                id="inputSnapshotName"
-                                name="inputSnapshotName"
-                                class="oui-input"
-                                data-ng-class="{ 'oui-input_error': nashaCustomSnapshotAddForm.inputSnapshotName.$invalid }"
-                                data-ng-model="NashaPartitionCustomSnapshotAddCtrl.snapshotName"
-                                data-ng-maxlength="NashaPartitionCustomSnapshotAddCtrl.options.snapshotName.maxLength"
-                                aria-describedby="snap-addon"
-                                required />
-                        </div>
-                    </div>
-                </div>
+        <oui-field data-label="{{::'nasha_custom_snapshot_modal_label_snapshot_name' | translate}}">
+            <div class="oui-input-group">
+                <button class="oui-button"
+                    disabled
```

Comment:
If we declare a button always disabled, i think it's not really a button, right?

Can we use a `span` to display the prefix?

Reply:
i cannot because it doesn't take the wanted style. See ui-kit oui-input-group doc.